components/speedreader/rust/ffi/Cargo.lock

```diff
@@ -270,10 +132,10 @@ version = "0.3.6"
 source = "git+https://github.com/AndriusA/rust-cssparser?branch=glibc#ad2f4d3c89e247ec1693a581ecace199e46f30fa"
```

Comment:
do we still need this?

Reply:
I removed it, but would like @antonok-edm to confirm it's ok. It seems like the only change is replacing the power function with a rust port of MUSL libc implementation? https://github.com/AndriusA/rust-cssparser/commit/ad2f4d3c89e247ec1693a581ecace199e46f30fa

Reply:
this was needed to prevent rust from linking with a newer version of libc on linux...

Reply:
as discussed in DM we need to keep that